### ISSUE TITLE
Adding a link to Libation in my desktop audiobook app

### ISSUE THREAD

**Issue by rstewa** [rmcrackan/Libation#914]
I have a desktop audiobook app that I created called [Audibly](https://github.com/rstewa/Audibly). I'd like to add a link to Libation in my app. Is this something you'd be cool with?

**Comment by rmcrackan** [comment]
Absolutely. Is it stable enough for me to [include in my FAQ](https://github.com/rmcrackan/Libation/blob/master/Documentation/FrequentlyAskedQuestions.md#q-now-that-ive-downloaded-my-books-how-can-i-listen-to-them)?

Thanks for filling this niche. Good audiobook-centric apps are hard to come by, esp for desktop.

**Comment by rmcrackan** [comment]
Ooooo, it's even in C# -- yummy. If I can find free time in my life, I may have to browse your repo for while.

**Comment by rstewa** [comment]
> Absolutely. Is it stable enough for me to [include in my FAQ](https://github.com/rmcrackan/Libation/blob/master/Documentation/FrequentlyAskedQuestions.md#q-now-that-ive-downloaded-my-books-how-can-i-listen-to-them)?
> 
> Thanks for filling this niche. Good audiobook-centric apps are hard to come by, esp for desktop.

I think so. It has ~3500 downloads and 30-40 daily active users right now from the microsoft store. I definitely need to add some unit tests but I've been lazy and haven't gotten around to it yet.

> Ooooo, it's even in C# -- yummy. If I can find free time in my life, I may have to browse your repo for while.

Please feel free. I'm a solo developer so I'd love to get some feedback on it.

**Comment by rmcrackan** [comment]
Added: https://github.com/rmcrackan/Libation/blob/master/Documentation/FrequentlyAskedQuestions.md#q-now-that-ive-downloaded-my-books-how-can-i-listen-to-them
